.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Preguntar sobre el encajamiento software para revisión
```

Comment:
It might be clearer to extract the verb "encajar" from the noun "encajamiento" (a.k.a. nominalization), and make it explicit that we are talking about rOpenSci's scope.

My revision:
```
name: Preguntar si el software encaja dentro del alcance de rOpenSci
```

(Instead of "encaja" I would also consider "está" -- which in itself is vague but combined with "dentro" feels precise and sound natural to my ear.)

Reply:
ok, change made, thank you!